### ISSUE TITLE
unable to install. Error related to mpixx

### ISSUE THREAD

**Issue by mudit-jain90** [PolyChord/PolyChordLite#109]
Hi,

I have been struggling with issues related to `commons', and unable to install Polychord. It's been over a month and a half, and I hope the developers can find some time to resolve this issue.

I have been trying to install Polychord on my Mac, but it spits out the following error:
￼
<img width="712" alt="Screenshot 2023-12-15 at 1 18 59 AM" src="https://github.com/PolyChord/PolyChordLite/assets/84209375/2cee1f80-e3b1-48ee-8aaf-bc080d74261b">

I'd be glad to receive any assistance please. 
Thanks! 
PS: I notice that there was another issue that got opened, related to commons. But looks like no progress has been made so far...

**Comment by yallup** [comment]
Are the instructions (updating gcc if you use brew) in:
https://github.com/PolyChord/PolyChordLite/pull/108/files 
Feasible to try? Updating compilers isn't ideal but I am running an up to date set on an M1 mac and don't have this issue so it's worth a try first if possible.

**Comment by mudit-jain90** [comment]
Thanks for the advice @yallup 
However this does not help me fix the issue. I get the same issue still. I have reinstalled gcc and even upgraded brew.

@williamjameshandley can you please reopen this ticket so that I can be assisted with this glitch I've been having for more than 3 months now? Thanks

**Comment by mudit-jain90** [comment]
HI. I think I was able to fix the error. I just had to reinstall mpich to fix the -commons issue.

Note: This reinstallation business of mpich did not work earlier. But somehow it did now. I don't know if it was due to the new iOS Sonoma 14.2.1 that I recently upgraded to, or due to reinstalling of gcc (as suggested by @yallup), or a combination of both. Or something else that I don't understand. Anyways, thought of sharing the fix here in case other people are struggling with the -commons error in Mac.
